srcpkgs/xfsprogs/template

```diff
@@ -1,24 +1,32 @@
 # Template file for 'xfsprogs'
 pkgname=xfsprogs
-version=6.5.0
+version=6.14.0
 revision=1
 build_style=gnu-configure
 configure_args="--enable-editline=yes --enable-lib64=no --libdir=/usr/lib
  --enable-lto"
-hostmakedepends="gettext"
+hostmakedepends="gettext pkg-config"
 makedepends="libblkid-devel libedit-devel inih-devel liburcu-devel
  libuuid-devel"
 short_desc="Utilities for managing the XFS filesystem"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="http://xfs.org/index.php/Main_Page"
 distfiles="${KERNEL_SITE}/utils/fs/xfs/xfsprogs/xfsprogs-${version}.tar.xz"
-checksum=8db81712b32756b97d89dd9a681ac5e325bbb75e585382cd4863fab7f9d021c6
+checksum=fa5ab77f8b5169ce48dd8de09446ad7e29834a05b8f52012bae411cf53ec1f58
 
 if [ "$CROSS_BUILD" ]; then
 	hostmakedepends+=" liburcu-devel"
 fi
 
+if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
```

Comment:
musl on archs other than `armv*` should have statx now

```suggestion
case "$XBPS_TARGET_MACHINE" in
	armv*-musl)
```

Reply:
the statx in new musl is still too old.

Reply:
ah ok